src/installation/guides/chroot-guide.md

```diff
@@ -0,0 +1,286 @@
+# Installation Guide (chroot)
+
+## Pre-installation
+
+### Set the keyboard layout
+
+The default console keymap is US. Available layouts can be listed with:
+
+```
+$ ls /usr/share/kbd/keymaps/*/*/*.map.gz | less
+```
+
+To modify the layout, append a corresponding file name to
+[loadkeys(1)](https://man.voidlinux.org/loadkeys), omitting path and file
+extension. For example, to set the Dvorak keyboard layout:
+
+```
+# loadkeys dvorak
+```
+
+### Verify the boot mode
+
+Many new motherboards will support booting in UEFI mode, but some will not,
+instead booting in legacy BIOS mode. You will need to know in which mode your
+system will boot when it is time to partition the disks.
+
+If you have boot via UEFI mode, the directory `/sys/firmware/efi` should exist
```

Comment:
When in UEFI mode, the directory /sys/firmware/efi should be present.  If you believe you should be in EFI mode and this directory is not present, verify that the efivars filesystem is mounted.